src/pages/home/home.scss

```diff
@@ -0,0 +1,73 @@
+.homeContainer {
+  height: 100%;
+
+  & .mainHeaderBackground {
+    background: url('../../assets/main-header-background.png') no-repeat;
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    position: absolute;
+    z-index: -1;
+  }
+
+  & .mainHeaderContent {
+    display: flex;
+    margin-left: 4em;
+
+    & div {
+      flex: 1;
+
+      & .header {
+        font-family: 'Fjalla One', sans-serif;
+        font-size: 4.25rem;
+        color: #ffffff;
```

Comment:
we need to use variables for the colors, I think @linux-nerd is going to add those variables in his current PR